bin/git-release

```diff
@@ -91,8 +91,8 @@ if test $# -gt 0; then
   git commit -a -m "$msg"
   # shellcheck disable=SC2086
   git tag $version -a -m "$msg" \
-    && git push $remote \
     && git push $remote --tags \
+    && git push $remote \
```

Comment:
IMHO, we could omit this line because `git push $remote --tags` already pushes the commit to remote branch.

Reply:
I don't know if that's the case, I'll run a quick test.

Reply:
I just did a test. The tag will be pushed, and you can find the commit if you reference it by SHA. However, the commit will **not** be pushed to the branch, so we still need both commands.

Reply:
@dominicbarnes 
My mistake. Push commit in this way doesn't push it to proper branch.